rename dashboard -> application

tonykploomber opened the pull request:
Rename dashboard -> application

Including text, image

neelasha23 commented on doc/apps/voila.md:
```diff
@@ -1,17 +1,17 @@
 ## Voilà
 
-Currently, Ploomber Cloud supports [Voilà](https://voila.readthedocs.io/en/stable/) for deploying Jupyter notebooks as dashboards. For information on how to write Voilà dashboards, [please check the documentation](https://voila.readthedocs.io/en/stable/).
+Currently, Ploomber Cloud supports [Voilà](https://voila.readthedocs.io/en/stable/) for deploying Jupyter notebooks as applications. For information on how to write Voilà applications, [please check the documentation](https://voila.readthedocs.io/en/stable/).
```
The same link is repeated.

tonykploomber replied:
[remove duplicated link](https://github.com/ploomber/doc/pull/68/commits/f9776ad3d828d1f1c72aa625fa093efe6fcb818b)

tonykploomber commented:
> 1. There are some mentions of `dashboard` in `user-guide/github.md`
> 2. [doc/static/copy-api-key.png](https://github.com/ploomber/doc/pull/68/files#diff-b5970bb8d15f1c030fbc2a71c566faeb899a56ad8885afee27dc5df1b110467c) In this image inputs should be
> 3. [doc/static/web-home.png](https://github.com/ploomber/doc/pull/68/files#diff-248a436db629b42241451e58a93f2cc91bde47c3701285c6d462211add39f844) : The latest homepage has `labels`, `type` etc which is missing from this image
> 
> @tonykploomber

1. Updated in [dashboard](https://github.com/ploomber/doc/pull/68/commits/93f42833afe8ab45ca82a608fb3b10914ef0256f)
2. Not sure what you meant, but I masked the api key and email in the image
3. Updated
[update image](https://github.com/ploomber/doc/pull/68/commits/d0a5281273c5f4e793b73e7e40f7c769f1c10fa1)

bryannho reviewed (commented):
lg. 
> Would you give me the file name of image? Not sure which one you are referring to 

I believe its `/doc/static/env-vars/env-vars-section.png`
@tonykploomber

neelasha23 commented:
> Not sure what you meant, but I masked the api key and email in the image 

I meant in the previous version there was a black bar on the inputs to hide the values. @tonykploomber

yafimvo commented:
> Would you give me the file name of image? Not sure which one you are referring to

It's this one
https://ploomber-doc--68.org.readthedocs.build/en/68/user-guide/env-vars.html

tonykploomber commented:
> > Not sure what you meant, but I masked the api key and email in the image
> 
> I meant in the previous version there was a black bar on the inputs to hide the values. @tonykploomber

Yes it's masked now

tonykploomber commented:
> update doc

[update doc](https://github.com/ploomber/doc/pull/68/commits/14a85b02939cd33879cbc6ecd47035d9ba162677)

tonykploomber commented:
> > Would you give me the file name of image? Not sure which one you are referring to
> 
> It's this one https://ploomber-doc--68.org.readthedocs.build/en/68/user-guide/env-vars.html

[update doc](https://github.com/ploomber/doc/pull/68/commits/14a85b02939cd33879cbc6ecd47035d9ba162677)

edublancas commented:
@neelasha23 @yafimvo @bryannho please review again and approve if all looks good
